site/content/overview/resource-validation.md

```diff
@@ -90,6 +90,7 @@ This step catches the following cases of invalid values:
 
 - Valid values from the Gateway API perspective but not supported by NGINX Gateway Fabric yet. For example, a feature in an HTTPRoute routing rule. For the list of supported features see [Gateway API Compatibility](gateway-api-compatibility.md) doc.
 - Valid values from the Gateway API perspective, but invalid for NGINX, because NGINX has stricter validation requirements for certain fields. These values will cause NGINX to fail to reload or operate erroneously.
+- Valid values from the Gateway API perspective, but invalid for NGINX Gateway Fabric, because such values can prevent generating NGINX configuration.
```

Comment:
we no longer need it, cause we removed this class of errors :)

Reply:
yes, updating